src/data/rat/basic.lean

```diff
@@ -0,0 +1,44 @@
+/-
+Copyright (c) 2019 Johannes Hölzl. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Johannes Hölzl, Mario Carneiro
+-/
+import algebra.euclidean_domain
+import data.int.cast
+import data.nat.gcd
+import data.rat.defs
+import logic.encodable.basic
+
+/-!
+# Basic lemmas for the Rational Numbers
+
+## Summary
+
+We put the (discrete) field structure on the type `ℚ` of rational numbers that
+was defined in `data.rat.defs`.
+
+## Main Definitions
+
+- `rat.field` is the field structure on `ℚ`.
+
+## Tags
```

Comment:
This could probably do with an `## Implementation notes` section explaining why this needs to be in a separate file.